doc/usage/configuration.rst

```diff
@@ -1841,7 +1841,17 @@ These options influence LaTeX output.
    ``'xelatex'`` or ``'lualatex'`` and making sure to use an OpenType font
    with wide-enough glyph coverage is often easier than trying to make
    ``'pdflatex'`` work with the extra Unicode characters.  Since Sphinx 2.0
-   the default is the GNU FreeFont which covers well Latin, Cyrillic and Greek.
+   the default is the GNU FreeFont which covers well Latin, Cyrillic and
+   Greek.
+
+   .. versionchanged:: 2.1.0
+
+      Use ``xelatex`` (and LaTeX package ``xeCJK``) by default for Chinese
+      documents.
+
+   .. versionchanged:: 2.2.1
```

Comment:
Now, we are using x.y.Z branch for urgent bugfix release. I think 2.0 branch is appropriate for this fix.

Reply:
Ah, sorry. I just read #6710 and understand this is a bug. Okay, go with 2.2.1.